### ISSUE TITLE
Do you ever read what's broken and fix it .. bags STILL don't open for bank or guild bank

### ISSUE THREAD

**Issue by BethVawter** [Jaliborc/Bagnon#660]
I still can't open my bank, or guild bank .. when I open them I get a random generic bag which is NOT my bank or guild bank.  I'm a programmer, I've tried all the fixes and none of them worked.  Do you ever fix your mistakes?

**Comment by Jaliborc** [comment]
What version are you running and what server are you on?

**Comment by Zippy4c** [comment]
I too am having issues seeing my Guild bank. I am using Version 7.3.11 and I play on Blackwater Raiders...I have an error message, I'm not sure if it will help but here it is... Date: 2018-02-15 14:18:57
ID: 2
Error occured in: Global
Count: 1
Message: ...Bagnon\external\Wildpants\features\tooltipCounts.lua line 73:
   attempt to index field '?' (a nil value)
Debug:
   ...Bagnon\external\Wildpants\features\tooltipCounts.lua:73:
      ...Bagnon\external\Wildpants\features\tooltipCounts.lua:57
   ...Bagnon\external\Wildpants\features\tooltipCounts.lua:135:
      ...Bagnon\external\Wildpants\features\tooltipCounts.lua:132
   [C]: ?
   [C]: ?
   [C]: ?
   ...AddOns\Auc-Advanced\Libs\LibExtraTip\LibExtraTip.lua:408: SetInventoryItem()
   ..\FrameXML\MainMenuBarBagButtons.lua:83: BagSlotButton_OnEnter()
   [string "*:OnEnter"]:1:
      [string "*:OnEnter"]:1
Locals:
tooltip = GameTooltip {
 0 = <userdata>
 SetQuestLogRewardSpell = <function> defined @Interface\AddOns\Auc-Advanced\Libs\LibExtraTip\LibExtraTip.lua:402
 SetSpellBookItem = <function> defined @Interface\AddOns\Auc-Advanced\Libs\LibExtraTip\LibExtraTip.lua:402
 SetCurrencyToken = <function> defined @Interface\AddOns\Altoholic\Tooltip.lua:680
 SetShapeshift = <function> defined @Interface\AddOns\Auc-Advanced\Libs\LibExtraTip\LibExtraTip.lua:402
 hasMoney = 1
 SetQuestRewardSpell = <function> defined @Interface\AddOns\Auc-Advanced\Libs\LibExtraTip\LibExtraTip.lua:402
 comparing = false
 Show = <function> defined =[C]:-1
 shownMoneyFrames = 1
 SetPetAction = <function> defined @Interface\AddOns\Auc-Advanced\Libs\LibExtraTip\LibExtraTip.lua:402
 SetUnitDebuff = <function> defined @Interface\AddOns\Auc-Advanced\Libs\LibExtraTip\LibExtraTip.lua:402
 SetLootItem = <function> defined @Interface\AddOns\Auc-Advanced\Libs\LibExtraTip\LibExtraTip.lua:402
 SetQuestItem = <function> defined =[C]:-1
 canAddRestStateLine = 1
 SetTrainerService = <function> defined @Interface\AddOns\Auc-Advanced\Libs\LibExtraTip\LibExtraTip.lua:402
 numMoneyFrames = 1
 SetInboxItem = <function> defined @Interface\AddOns\Auc-Advanced\Libs\LibExtraTip\LibExtraTip.lua:402
 SetBuybackItem = <function> defined @Interface\AddOns\Auc-Advanced\Libs\LibExtraTip\LibExtraTip.lua:402
 shoppingTooltips = <table> {
 }
 SetMerchantItem = <function> defined @Interface\AddOns\Auc-Advanced\Libs\LibExtraTip\LibExtraTip.lua:402
 SetQuestLogItem = <function> defined =[C]:-1
 LibExtraTipRight = <table> {
 }
 SetUnitBuff = <function> defined @Interface\AddOns\Auc-Advanced\Libs\LibExtraTip\LibExtraTip.lua:402
 SetTalent = <function> defined @Interface\AddOns\Auc-Advanced\Libs\LibExtraTip\LibExtraTip.lua:402
 SetTradePlayerItem = <function> defined @Interface\AddOns\Auc-Advanced\Libs\LibExtraTip\LibExtraTip.lua:402
 LibExtraTipLeft = <table> {
 }
 SetSendMailItem = <function> defined @Interface\AddOns\Auc-Advanced\Libs\LibExtraTip\LibExtraTip.lua:402
 __tamedCounts = false
 updateTooltip = 0.079999991506338
 SetInventoryItem = <function> defined @Interface\AddOns\Auc-Advanced\Libs\LibExtraTip\LibExtraTip.lua:402
 SetAuctionItem = <function> defined @Interface\AddOns\Auc-Advanced\Libs\LibExtraTip\LibExtraTip.lua:402
 SetLootRollItem = <function> defined @Interface\AddOns\Auc-Advanced\Libs\LibExtraTip\LibExtraTip.lua:402
 SetTradeTargetItem = <function> defined @Interface\AddOns\Auc-Advanced\Libs\LibExtraTip\LibExtraTip.lua:402
 SetAuctionSellItem = <function> defined @Interface\AddOns\Auc-Advanced\Libs\LibExtraTip\LibExtraTip.lua:402
 SetBagItem = <function> defined @Interface\AddOns\Auc-Advanced\Libs\LibExtraTip\LibExtraTip.lua:402
 needsReset = true
 SetRecipeReagentItem = <function> defined =[C]:-1
 SetGuildBankItem = <function> defined @Interface\AddOns\Auc-Advanced\Libs\LibExtraTip\LibExtraTip.lua:402
 SetHyperlink = <function> defined =[C]:-1
 SetRecipeResultItem = <function> defined =[C]:-1
}
link = "[Netherweave Bag]"
itemID = 21841
players = 1
total = 5
(for generator) = <function> defined @Interface\AddOns\Bagnon\external\Wildpants\libs\LibItemCache-2.0\LibItemCache-2.0.lua:162
(for state) = nil
(for control) = "® UnZipped"
owner = "® UnZipped"
info = <table> {
 isguild = true
 name = "UnZipped"
 faction = "Alliance"
 cached = false
 realm = "Blackwater Raiders"
}
color = "%s"
(*temporary) = nil
(*temporary) = <table> {
 isguild = true
 name = "UnZipped"
 faction = "Alliance"
 cached = false
 realm = "Blackwater Raiders"
}
(*temporary) = <table> {
 GetRGB = <function> defined @Interface\SharedXML\Util.lua:594
 OnLoad = <function> defined @Interface\SharedXML\Util.lua:583
 b = 0
 WrapTextInColorCode = <function> defined @Interface\SharedXML\Util.lua:625
 GenerateHexColor = <function> defined @Interface\SharedXML\Util.lua:621
 GetRGBA = <function> defined @Interface\SharedXML\Util.lua:602
 IsEqualTo = <function> defined @Interface\SharedXML\Util.lua:587
 g = 0.64
 r = 0.77
 SetRGBA = <function> defined 
AddOns:
  Swatter, v7.5.5724 (TasmanianThylacine)
  Altoholic, vr183
  AskMrRobot, v57
  AucAdvanced, v7.5.5724 (TasmanianThylacine)
  AucFilterBasic, v7.5.5724 (TasmanianThylacine)
  AucFilterOutlier, v7.5.5724.5459(7.5/embedded)
  AucMatchUndercut, v7.5.5724.5585(7.5/embedded)
  AucScanData, v7.5.5724 (TasmanianThylacine)
  AucStatHistogram, v7.5.5724 (TasmanianThylacine)
  AucStatiLevel, v7.5.5724 (TasmanianThylacine)
  AucStatPurchased, v7.5.5724 (TasmanianThylacine)
  AucStatSales, v7.5.5724.5598(7.5/embedded)
  AucStatSimple, v7.5.5724 (TasmanianThylacine)
  AucStatStdDev, v7.5.5724 (TasmanianThylacine)
  AucStatWOWEcon, v7.5.5724.5613(7.5/embedded)
  AucUtilAHWindowControl, v7.5.5724.5575(7.5/embedded)
  AucUtilAppraiser, v7.5.5724.5719(7.5/embedded)
  AucUtilAskPrice, v7.5.5724.5599(7.5/embedded)
  AucUtilAutoMagic, v7.5.5724.5686(7.5/embedded)
  AucUtilCompactUI, v7.5.5724.5715(7.5/embedded)
  AucUtilEasyBuyout, v7.5.5724.5576(7.5/embedded)
  AucUtilFixAH, v7.5.5724 (TasmanianThylacine)
  AucUtilItemSuggest, v7.5.5724.5651(7.5/embedded)
  AucUtilPriceLevel, v7.5.5724.5633(7.5/embedded)
  AucUtilScanButton, v7.5.5724.5631(7.5/embedded)
  AucUtilScanFinish, v7.5.5724.5720(7.5/embedded)
  AucUtilScanProgress, v7.5.5724.4979(7.5/embedded)
  AucUtilScanStart, v7.5.5724.5347(7.5/embedded)
  AucUtilSearchUI, v7.5.5724.5721(7.5/embedded)
  AucUtilSimpleAuction, v7.5.5724.5705(7.5/embedded)
  AucUtilVendMarkup, v7.5.5724.4828(7.5/embedded)
  Babylonian, v5.1.DEV.332(/embedded)
  BagBrother, v
  Bagnon, v7.3.11
  BagnonGuildBank, v7.3.11
  BattlePetBreedID, vr166
  BattlePetQualityNotifier, v1.3.0
  BeanCounter, v7.5.5724 (TasmanianThylacine)
  BlockChinese, vv1.5
  Carbonite, v7.3.0
  CollectMe, v2.9.6
  Configator, v5.1.DEV.431(/embedded)
  DataStore, vr64
  DataStoreAchievements, vr46
  DataStoreAgenda, vr26
  DataStoreAuctions, vr49
  DataStoreCharacters, vr41
  DataStoreContainers, vr57
  DataStoreCrafts, vr78
  DataStoreCurrencies, vr36
  DataStoreGarrisons, vr22
  DataStoreInventory, vr51
  DataStoreMails, vr50
  DataStorePets, vr43
  DataStoreQuests, vr43
  DataStoreReputations, vr41
  DataStoreSpells, vr31
  DataStoreStats, vr30
  DataStoreTalents, vr60
  DBMCore, v7.3.21
  DBMDefaultSkin, v
  DBMStatusBarTimers, v
  DebugLib, v5.1.DEV.337(/embedded)
  EasyObliterate, v30
  Enchantrix, v7.5.5724 (TasmanianThylacine)
  EnchantrixBarker, v7.5.5724 (TasmanianThylacine)
  GarrisonCommanderBroker, v2.18.5 1.1.0
  GTFO, v4.44.3
  GuildCheck, vv 2.5.0
  GuildNoteChanges, vv1.2.3
  GuildTracker, v7.0.3-1
  HealBot, v7.3.2.0
  HealBotTips, v7.3.2.0
  Informant, v7.5.5724 (TasmanianThylacine)
  LibExtraTip, v5.12.DEV.430(/embedded)
  LucidNightmareHelper, v
  Overachiever, v0.99.10
  OverachieverTabs, v0.99.10
  Postal, vv3.5.8
  QuestCompletist, v109.5
  Rematch, v4.7.11
  Skada, v1.7.2
  SlideBar, v7.5.5724 (TasmanianThylacine)
  SpellStealer, v1.85
  Stubby, v7.5.5724 (TasmanianThylacine)
  TipHelper, v5.12.DEV.405(/embedded)
  Titan, v5.12.6.70300
  TitanBag, v5.12.6.70300
  TitanClock, v5.12.6.70300
  TitanGold, v5.12.6.70300
  TitanLocation, v5.12.6.70300
  TitanLootType, v5.12.6.70300
  TitanPerformance, v5.12.6.70300
  TitanRepair, v5.12.6.70300
  TitanVolume, v5.12.6.70300
  TitanXP, v5.12.6.70300
  WhoPulled, v1.0.14
  WorldQuestTracker, v
  BlizRuntimeLib_enUS v7.3.5.70300 <none>
  (ck=d0e)

**Comment by Torsin** [comment]
@Zippy4c does it still error if you disable Auctioneer (by unchecking Stubby which is the dependency for Aucitoneer)?

**Comment by Zippy4c** [comment]
I'll make sure to try that next time I log in...thank you

**Comment by Zippy4c** [comment]
That did not work...still cannot see anything in guild bank. Just empty squares

**Comment by Torsin** [comment]
@Zippy4c with Auctioneer disabled, what error do you get?  I hate to say it, but if only Bagnon (or with swatter as well) is enabled does it still not work/give an error.

**Comment by Zippy4c** [comment]
When I open up the guild bank I do not get an error message. Swatter catches some bugs and I thought it would be helpful to you. But obviously that is not the case. I disabled the Guild Bank Option in Bagnon and now I can see the Blizzard UI version so at least I can see what's in the bank now. No other addons are affected. I also disabled swatter and then tried to see guild bank but that didn't work either... Maybe someday an update will fix it...
